docs/troubleshoot-ksql.rst

```diff
@@ -0,0 +1,306 @@
+.. _troubleshoot-ksql:

```

Comment:
this file must be referenced from a parent file via `toctree` or it won't be displayed in the lefthand navigation (and won't have a breadcrumb trail etc).